### PR TITLE
Feature/issue 85 view figures

### LINKED ISSUE #85: View
**Priority :**
1

**Duration :** 
0.5

**Description:**
Creation of the figures view to display the list of figures in the form of a table. 

**File added / modified:**

**Additional context**

Add any other context or screenshots here.

### PR DESCRIPTION
**Pull request by vanmarcke**
Issue [#85]

Creation of the figures view to display the list of figures in the form of a table

### CODE REVIEW

File: src/Controller/FigureController.php [hunk 1]
```diff
@@ -16,6 +16,17 @@
 
 class FigureController extends AbstractController
 {
+    #[Route('/figures/list', name: 'figures_list')]
+    public function figuresList(FigureManagerInterface $figureManager): Response
+    {
+        $figuresList = $figureManager->getListFigures();
```
Comment (antoinetrouve) [inline src/Controller/FigureController.php:22]:
```suggestion
        $figureList = $figureManager->getListFigures();
```

File: src/Manager/FigureManager.php [hunk 2]
```diff
@@ -20,6 +20,16 @@ public function __construct(private SnowFigureRepository $snowFigureRepository,
         $this->slugger = $slugger;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getListFigures(): array
```
Comment (antoinetrouve) [inline src/Manager/FigureManager.php:26]:
```suggestion
    public function getFigureList(): array
```

**Review by antoinetrouve — APPROVED** [review]

File: src/Controller/FigureController.php (hunk 1 above)
Comment (vanmarcke) [reply src/Controller/FigureController.php:22]: Done

File: src/Manager/FigureManager.php (hunk 2 above)
Comment (vanmarcke) [reply src/Manager/FigureManager.php:26]: Done

**Pull request merged**